### PR TITLE
fix: teams stats doesn't aggregate from latest epoch

### LINKED ISSUE #10722: Team API aggregation epooch has wrong startpoint
If you query teamStatistics on testnet for this team (the incentive winner).

```
query teamStats{
  teamsStatistics(teamId:"0c7b592e553af7d5204bff22d22bea16cf6d11f7555615297f2d9559538a14f8",aggregationEpochs:30){
    edges{
      node{
        teamId
        totalQuantumVolume
        totalQuantumRewards
        totalGamesPlayed
        gamesPlayed
        quantumRewards{
          epoch
          totalQuantumRewards
        }
      }
    }
  }
}
```

You will get all of these results ... 30 epochs worth of results .... and if you add up the reward per epoch i asume you will get the number at the top.

```
{
  "data": {
    "teamsStatistics": {
      "edges": [
        {
          "node": {
            "teamId": "0c7b592e553af7d5204bff22d22bea16cf6d11f7555615297f2d9559538a14f8",
            "totalQuantumVolume": "0",
            "totalQuantumRewards": "18453",
            "totalGamesPlayed": 5,
            "gamesPlayed": [
              "0798fd6ac2fa0c3b1ecb150d9a7d112890f9d9fea3374c918795443eef380f06",
              "0e08dcff9a35c0f92caa301cd80036403e1467834263b60fe7e4c8a540d5eb6d",
              "21f16328fa7a2253bda78147fe766aa22f723dd51c2d7a0d90c185294352e7ca",
              "541048fb3f95dcbd6d53ec639dfcb6eec75c1bff042ac51fd016b8d4230de148",
              "bf6e4217599e0846309a84d4a47ea8aa4404bf86d05052f8caf9a4f8e8ba1061"
            ],
            "quantumRewards": [
              {
                "epoch": 12604,
                "totalQuantumRewards": "775"
              },
              {
                "epoch": 12605,
                "totalQuantumRewards": "734"
              },
              {
                "epoch": 12606,
                "totalQuantumRewards": "759"
              },
              {
                "epoch": 12607,
                "totalQuantumRewards": "710"
              },
              {
                "epoch": 12608,
                "totalQuantumRewards": "605"
              },
              {
                "epoch": 12609,
                "totalQuantumRewards": "736"
              },
              {
                "epoch": 12610,
                "totalQuantumRewards": "259"
              },
              {
                "epoch": 12611,
                "totalQuantumRewards": "812"
              },
              {
                "epoch": 12612,
                "totalQuantumRewards": "736"
              },
              {
                "epoch": 12613,
                "totalQuantumRewards": "605"
              },
              {
                "epoch": 12614,
                "totalQuantumRewards": "606"
              },
              {
                "epoch": 12615,
                "totalQuantumRewards": "740"
              },
              {
                "epoch": 12616,
                "totalQuantumRewards": "180"
              },
              {
                "epoch": 12617,
                "totalQuantumRewards": "757"
              },
              {
                "epoch": 12618,
                "totalQuantumRewards": "710"
              },
              {
                "epoch": 12619,
                "totalQuantumRewards": "711"
              },
              {
                "epoch": 12620,
                "totalQuantumRewards": "748"
              },
              {
                "epoch": 12621,
                "totalQuantumRewards": "111"
              },
              {
                "epoch": 12622,
                "totalQuantumRewards": "115"
              },
              {
                "epoch": 12623,
                "totalQuantumRewards": "29"
              },
              {
                "epoch": 12624,
                "totalQuantumRewards": "859"
              },
              {
                "epoch": 12625,
                "totalQuantumRewards": "808"
              },
              {
                "epoch": 12626,
                "totalQuantumRewards": "786"
              },
              {
                "epoch": 12627,
                "totalQuantumRewards": "739"
              },
              {
                "epoch": 12628,
                "totalQuantumRewards": "739"
              },
              {
                "epoch": 12629,
                "totalQuantumRewards": "811"
              },
              {
                "epoch": 12630,
                "totalQuantumRewards": "742"
              },
              {
                "epoch": 12631,
                "totalQuantumRewards": "606"
              },
              {
                "epoch": 12632,
                "totalQuantumRewards": "712"
              },
              {
                "epoch": 12633,
                "totalQuantumRewards": "213"
              }
            ]
          }
        }
      ]
    }
  }
}
```

BUT ... the current epoch is 12666! 

So the aggregationEpochs is giving you the last 30 epochs _that have results_ .... not the last 30 epochs overall.  You will see it is going from 12633 to 12604.

## Solution

Adjust aggregationEpochs treatment so it always starts from the _current_ epoch and counts back the number of epochs given.

## Extra Note

We also use aggregationEpochs on the referral APIs ... and if this treatment is the same there that will also be wrong ... so please check.

### PR DESCRIPTION
**Pull request by guoguojin**
Closes #10722

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased 0.75.0
+## Unreleased 0.74.4
```
Comment (jeremyletang) [inline CHANGELOG.md:3]: Not sure why the change here.

Comment (guoguojin) [reply CHANGELOG.md:3]: I wasn't sure, but seeing as you released the 0.74.3 tag yesterday, I thought we had to bump it, I will revert it.

File: datanode/sqlstore/migrations/0091_epochs_add_start_and_end_block_height.sql [hunk 2]
```diff
@@ -24,7 +24,7 @@ BEGIN
     REFRESH MATERIALIZED VIEW CONCURRENTLY current_epochs;
     RETURN NULL;
 END;
-$$
+$$;
```
[the remark this answers is not in the document]
Comment (guoguojin) [reply datanode/sqlstore/migrations/0091_epochs_add_start_and_end_block_height.sql:27]: The migration failed when I ran it on my computer using goose up/down. Not sure why, could have been something wrong on my computer, I will double check.

[the remark this answers is not in the document]
Comment (guoguojin) [reply datanode/sqlstore/migrations/0091_epochs_add_start_and_end_block_height.sql]: It appears to only fail when you run the migrations manually using the goose CLI so I have reverted for now.

**Review by peterbarrow — APPROVED** [review]
